src/mat_properties.py

```diff
@@ -0,0 +1,21 @@
+import materials_dict_master.py
+
+
+def mat_properties(materials)
+""" This will generate a nested dictionary for the materials and important material properties that will be used in other functions. Material information will be stored in an external file that is read in using this function.
+
+inputs
+materials: The materials that will be used in the particular run of the program.
+outputs
+material_property_library: dictionary containing the material properties for each material to be modeled.
+"""
+
+   
+    material_property_library = {'materials[0]': {'rho': materials_dict_master['material[0]']['rho'], 'k': materials_dict_master['material[0]']['k'], 'c': materials_dict_master['material[0]']['c'] },
```

Comment:
Not sure what you're aiming for with these keys.... but this looks like it will largely recreate a nearly identical dictionary to the one you are importing.  reason?

Reply:
I believe in the case that we wanted to add different materials to the master dictionary you would need to use this function to build material_property_library, correct? So long as there are 5 materials chosen.